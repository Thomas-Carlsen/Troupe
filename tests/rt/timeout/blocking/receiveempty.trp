(* this test blocks because there are no sends *)
receive []
