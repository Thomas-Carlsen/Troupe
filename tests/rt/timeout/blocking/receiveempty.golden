2019-03-04T15:47:10.391Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
